spec/features/adventures/user_edits_adventure_spec.rb

```diff
@@ -51,24 +43,15 @@
   scenario "authenticated user successfully edits an adventure's bucket list" do
     user = FactoryGirl.create(:user)
     visit new_user_session_path
-    fill_in "Email", with: user.email
-    fill_in "Password", with: user.password
-    click_button "Log in"
-    bucket_list_1 = FactoryGirl.create(
-      :bucket_list,
-      title: "Australia",
-      user_id: user.id
-    )
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.